packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -170,18 +170,18 @@ export function signInActor({ services }: SignInMachineOptions) {
                 },
               },
             },
-            resolved: { always: '#signInActor.preResolved' },
+            resolved: { always: '#signInActor.updateCurrentUser' },
             rejected: { always: '#signInActor.rejected' },
           },
         },
         autoSignIn: {
-          initial: 'pending',
+          initial: 'signIn',
```

Comment:
Yes we effectively changed the way the machine worked.

Previously - auto signin would start in the `pending` state and wait for the Authhub event to fire the `signIn` event and the `autoSignIn` Authenticator machine event. This would then trigger the signin flow.

Now - the confirmSignIn api call will return a next step of `signIn` which will indicate that we should use the `autoSignIn` API that is exported from Auth. (In a previous design review Israel had the autoSignin function returned as a callback but after API review that was moved to an export).

Reply:
Do we still use the `pending` state?

Reply:
Some of the signin machine actor substates having a `pending` state. This pending state referenced here specifically referencing was the `autoSignin#pending` state which is now removed